### ISSUE TITLE
Win/Loss Incrementing

### ISSUE THREAD

**Issue by dschnaars** [Leek727/ChessSchedule#29]
Currently the moderator can select "Won" or "Lost" multiple times in a given round and the result is a multiple incrementing of the wins/losses for both players involved in the pairing. A mis-click of this nature can also not be un-done, meaning win/loss records and ELOs are irreparably changed.

**Comment by clayrock78** [comment]
I propose a fix that involves changing the way that scoring works as a whole. Each time a new round of matches is started, we will create a collection of game result objects. The changes in win/loss record and ELO that are applied to each player will only be made at the time that the next_round event is used, or the game is ended. This way, it will be much easier to prevent multiple different game results from being entered for one pairing.

Additionally, I believe that admin-entered results should have their own unique functionality rather than building off what exists, as is the case right now.

Let me know what you think @PhantomLel

**Comment by PhantomLel** [comment]
Yeah that sounds fine. You should get started on that but I still am going to finish rewriting in Svelte first
